pkg/kubelet/cm/cpumanager/cpu_assignment.go

```diff
@@ -190,7 +190,7 @@ func takeByTopology(topo *topology.CPUTopology, availableCPUs cpuset.CPUSet, num
 	//    on the same sockets as the whole cores we have already taken in this
 	//    allocation.
 	for _, c := range acc.freeCPUs() {
-		klog.V(4).Infof("[cpumanager] takeByTopology: claiming CPU [%d]", c)
+		klog.V(4).InfoS("takeByTopology: claiming CPU", "CPU", c)
```

Comment:
```suggestion
		klog.V(4).InfoS("takeByTopology: claiming CPU", "cpu", c)
```

Reply:
Use `cpu` instead now. Could you review it all again, pls?@ehashman